I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 302, in script
    Shizuku "Quero ficar fora disso. Se tiver algo com que precise de ajuda, faça um pedido e não me incomode mais. Meu poder, para você saber, é me comunicar com os mortos."
Exception: Sayer 'Shizuku' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 302, in script
    Shizuku "Quero ficar fora disso. Se tiver algo com que precise de ajuda, faça um pedido e não me incomode mais. Meu poder, para você saber, é me comunicar com os mortos."
  File "/Users/alifoo/Desktop/renpy-8.3.7-sdk/renpy/ast.py", line 2586, in execute
    Say.execute(self)
  File "/Users/alifoo/Desktop/renpy-8.3.7-sdk/renpy/ast.py", line 583, in execute
    who = eval_who(self.who, self.who_fast)
  File "/Users/alifoo/Desktop/renpy-8.3.7-sdk/renpy/ast.py", line 472, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'Shizuku' is not defined.

macOS-15.4.1-arm64-arm-64bit arm64
Ren'Py 8.3.7.25031702
vnmath 1.0
Sun May 18 21:51:44 2025
